.changeset/calm-islands-beam.md

```
@@ -0,0 +1,22 @@
+---
+'mastracode': minor
+---
+
+Added environment-controlled process memory diagnostics and the `/profile` command for the TUI and headless CLI.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)